reporter/src/main/kotlin/reporters/CycloneDxReporter.kt

```diff
@@ -157,7 +169,7 @@ class CycloneDxReporter : Reporter {
                 group = pkg.id.namespace
                 name = pkg.id.name
                 version = pkg.id.version
-                description = pkg.description
+                description = escapeXml(pkg.description)
```

Comment:
Consider using `org.apache.commons.lang.StringEscapeUtils` instead.

Reply:
We don't have a direct dependency on Apache Commons in the reporter module yet, and I'd prefer to keep it like that. Also, please note that this is only a work-around until https://github.com/CycloneDX/cyclonedx-core-java/issues/38 is fixed.

Reply:
> We don't have a direct dependency

I'm aware of that, but if should be using this argument more frequently in future then we should re-consider adding it...but now it is fine.